iron-fit-behavior.html

```diff
@@ -253,11 +253,13 @@
       if (typeof this._isRTL === 'undefined') {
         this._isRTL = window.getComputedStyle(this).direction == 'rtl';
       }
-          
       this.positionTarget = this.positionTarget || this._defaultPositionTarget;
       if (this.autoFitOnAttach) {
-        if (window.getComputedStyle(this).display === 'none') {
-          setTimeout(function() {
+        // If invisible or in a shadow root that's still not connected, wait a tick.
+        if (window.getComputedStyle(this).display === 'none' ||
+          document.body.contains(this) === false) {
```

Comment:
This is a different behavior compared to 1.x - awaiting news on https://github.com/webcomponents/shadydom/issues/120